doc/articles/uno-publishing-windows.md

```diff
@@ -0,0 +1,100 @@
+---
+uid: uno.publishing.windows
+---
+
+# Publishing Your App for Windows
+
+## Preparing For Publish
+
+## Building your app
+
+Building your app requires using the msbuild command (`dotnet build` is not compatible as of WinAppSDK 1.5).
+
+### Side loading (without signing)
+
+#### Package the unsigned app
+
+Packaging the app without a code signature allows for installing the app on a machine without having to install the signer's certificate. This portion of the guide is derived from the [official Windows App SDK documentation](https://learn.microsoft.com/en-us/windows/msix/package/unsigned-package).
+
+To package your app:
+
+- Update the `Package.appxmanifest` file with the following `Identity` node to include the following `OID`:
+
+  ```xml
+  <Identity Name="MyApp"
+            Publisher="CN=AppModelSamples, OID.2.25.311729368913984317654407730594956997722=1"
+            Version="1.0.0.0" />
+  ```
+
+- Build your app in the folder of the app's `.csproj` by using the following command:
+
+    ```pwsh
+    msbuild /r /p:TargetFramework=net8.0-windows10.0.19041 /p:Configuration=Release /p:Platform=x64 /p:GenerateAppxPackageOnBuild=true /p:AppxBundle=Never /p:UapAppxPackageBuildMode=Sideloading /p:AppxPackageDir="C:/temp/output/" /p:AppxPackageSigningEnabled=false
```

Comment:
This should also include docs on how to make a bundle of x64, ARM64 and x86 packages (for Store publishing)